### PR TITLE
Removed Static Variable from rzasm

### PR DESCRIPTION
**Pull request by HN026**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

...

**Test plan**

...

**Closing issues**

Partially addresses https://github.com/rizinorg/rizin/issues/4055

### CODE REVIEW

File: librz/include/rz_asm.h [hunk 1]
```diff
@@ -124,6 +125,10 @@ typedef struct rz_asm_t {
 	HtPP *flags;
 	int seggrn;
 	bool pseudo;
+	RzPVector *pvec;
```
Comment (XVilka) [inline librz/include/rz_asm.h]: Please check how it's done in https://github.com/rizinorg/rizin/commit/a592ea1e8c503e663b37e58c4beccecb40d2cb5e

See also this:
```
Missing type comment at <rizin/librz/include/rz_asm.h:128:13> (token is not a comment)
Error: Process completed with exit code 1.
```

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/include/rz_il/rz_il_opbuilder_begin.h [hunk 2]
```diff
@@ -35,6 +35,9 @@
 
 #include <rz_il/rz_il_opcodes.h>
 
+#undef AND
+#undef SIGNED
+
```
Comment (wargio) [inline librz/include/rz_il/rz_il_opbuilder_begin.h]: these should go in the rz_il_opbuilder_end

File: librz/include/rz_asm.h (hunk 1 above)
Comment (HN026) [reply librz/include/rz_asm.h]: Okay I will

File: librz/include/rz_il/rz_il_opbuilder_begin.h (hunk 2 above)
Comment (HN026) [reply librz/include/rz_il/rz_il_opbuilder_begin.h]: Okay

**Review by XVilka — CHANGES_REQUESTED** [review]
What I mean is using plugin-specific context instead, just like it's done in Tricore plugin

**Comment by HN026** [comment]
> What I mean is using plugin-specific context instead, just like it's done in Tricore plugin

Okay Sure

File: librz/include/rz_asm.h [hunk 3]
```diff
@@ -12,6 +12,7 @@
 #include <rz_parse.h>
 #include <rz_bind.h>
 #include <rz_config.h>
+#include "../asm/arch/include/disas-asm.h"
```
Comment (wargio) [inline librz/include/rz_asm.h]: no. this cannot be here.

Comment (HN026) [reply librz/include/rz_asm.h]: Okay

**Review by XVilka — APPROVED** [review]
Yes, that is the way of doing it. Thank you!

**Comment by XVilka** [comment]
@HN026 next time don't forget to refer to the corresponding issue in the PR body. If it fully solves it "Closes #XXX" if only partially - "Partially addresses #XXX" so that GitHub performs necessary chores automatically.

**Comment by HN026** [comment]
> @HN026 next time don't forget to refer to the corresponding issue in the PR body. If it fully solves it "Closes #XXX" if only partially - "Partially addresses #XXX" so that GitHub performs necessary chores automatically.

Okayyyy

**Pull request merged**